src/store/sessions-slice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+let accountsInLocalStorage = JSON.parse(localStorage.getItem("sessions"));
+let isAnAccountLoggedIn = JSON.parse(localStorage.getItem("isAnAccountLogged"))
+
+export const sessionsSlice = createSlice({
+    name: "sessions",
+    initialState: {
+        accountsLogged: accountsInLocalStorage ? accountsInLocalStorage : [],
+        isAnAccountLogged: isAnAccountLoggedIn ? true : false,
+    },
+    reducers: {
+        addNewAccount(state, action) {
+            state.accountsLogged.push(action.payload);
+            localStorage.setItem("sessions", JSON.stringify(state.accountsLogged));
```

Comment:
I believe there's an npm package (redux middleware) that let you choose part of the state in redux and save it to local storage after an update, use that package ;)